app/helpers/practices_helper.rb

```diff
@@ -97,4 +97,9 @@ def get_tool_resource_text(resource_type)
     }
     content[resource_type.to_sym]
   end
+
+  def sort_adoptions_by_state_and_station_name(adoptions)
+    adoptions.exclude_clinical_resource_hubs.sort_by { |a| fac = a.va_facility; [fac.street_address_state, fac.official_station_name.downcase] } +
+      adoptions.exclude_va_facilities.sort_by { |a| a.clinical_resource_hub.official_station_name.downcase[/\d+/].to_i }
```

Comment:
nit: we can do  `a.clinical_resource_hub.visn.number` just in case we ever decide to remove the VISN number from the name

Reply:
Makes sense. It has now been refactored!